compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -181,6 +181,10 @@ impl<'tcx> MonoItem<'tcx> {
         }
         .map(|hir_id| tcx.hir().span(hir_id))
     }
+
+    pub fn codegen_dep_node(&self, tcx: TyCtxt<'tcx>) -> DepNode {
```

Comment:
Nit: I would leave a comment here explaining this is only used by cg_clif

Reply:
Done